fenix/app/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <dimen name="tabs_strip_height">48dp</dimen>
```

Comment:
We need this so on phones, this is 0dp. This needs to be set in xml layouts so the size is set by the parent. We don't want to set the height in compose (which is currently defining the UI but the parent is still View which has the constraints, margins and the size)

Reply:
won't it still be marked as `GONE` though because of the logic with the `tablet` boolean dimen?

Reply:
You're right there. However, moving forward, as we enable it by default the check in many places will change from `isTabletAndTabStripEnabled` to `isTablet`. Now `isTablet` is simply sw600dp check. We can simplify that by having 2 `tabs_strip_height` dimensions for phones and tablets. Simplifying the marging upate calls to one lines instead of if (tablet ) 48 else 0.